uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/bookingrequest/BookingRequestView.kt

```diff
@@ -234,13 +236,15 @@ class BookingRequestView @JvmOverloads constructor(context: Context,
     }
 
     override fun onLoadingButtonClick() {
-         if (bookingRequestPassengerDetailsWidget.findAndfocusFirstInvalid() || !focusPaymentIfNotSet()) {
-             bookingRequestButton.onLoadingComplete()
-         } else {
-             hideSoftKeyboard()
-             presenter.makeBooking()
-             cancelButton.isEnabled = false
-         }
+        if (KarhooUISDKConfigurationProvider.configuration
```

Comment:
I suggest to make the condition a bit more readable like : ( you can create two small private methods in the same file)
if (!isKarhooUser() && !formIsValid()) {
}